### PR TITLE
fix(prometheus): collect nginx_status and return the body

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by Yiyiyimu — APPROVED** [review]
Manually test and it works

File: t/cli/test_prometheus.sh [hunk 1]
```diff
@@ -35,6 +35,11 @@ if [ ! $code -eq 200 ]; then
     exit 1
 fi
 
+if ! curl -i http://127.0.0.1:9091/apisix/prometheus/metrics | grep "apisix_nginx_http_current_connections" > /dev/null; then
+    echo "failed: should listen default prometheus address"
```
Comment (jbampton) [inline t/cli/test_prometheus.sh:39]:
```suggestion
    echo "failed: should listen at default prometheus address"
```

File: t/cli/test_prometheus.sh [hunk 2]
```diff
@@ -35,6 +35,11 @@ if [ ! $code -eq 200 ]; then
...
+    exit 1
+fi
+
 make stop
 
 echo "passed: should listen default prometheus address"
```
Comment (jbampton) [inline t/cli/test_prometheus.sh:45]:
```suggestion
echo "passed: should listen at default prometheus address"
```

Comment (jbampton) [reply t/cli/test_prometheus.sh:45]: Some more in this file further down I think should be the same for better grammar like for:

`should listen at ....`

Comment (spacewander) [reply t/cli/test_prometheus.sh]: @jbampton 
Applied. Thanks!

**Review by tokers — APPROVED** [review]

**Pull request merged**